pages/docs/user-docs/action-build.md

```diff
@@ -1,303 +1,185 @@
 ---
-title: Build an Image
+title: Build a Container
 sidebar: user_docs
 permalink: docs-build
 folder: docs
+toc: false
 ---
 
-Build is the process where we install an operating system and then configure it appropriately for a specified need. To do this we use a [Singularity recipe](/docs-recipes) file (a text file called `Singularity`) which is a recipe of how to specifically build the container. Here we will overview the usage, best practices, and quick examples.
+The `build` command lets you obtain a container. 
 
+`build` is the "Swiss army knife" of container creation.  You can use it to download and assemble pre-built containers from external resources like [Singularity Hub](https://singularity-hub.org/) and [Docker Hub](https://hub.docker.com/).  You can use it to convert containers between the various formats supported by Singularity.  And you can use it in conjunction with a [Singularity Recipe](/docs-recipes) file to create a container from scratch and customized it to fit your needs. 
 
-**Getting Started**
-For a detailed **walkhrough of getting started** with build, we will point you to the <a href="/quickstart">quickstart</a>. 
+{% include toc.html %}
 
-**Build Environment**
-If you want to customize the cache, specify Docker credentials, or any custom tweaks to your build environment, see <a href="/build-environment">build environment</a>. 
+## Overview
```

Comment:
This needs to stay - all of these variables are essential to know, most relevant for build, and it's not a big enough section to warrant it's own thing. Let's add this guy back!